README.md

```diff
@@ -29,8 +29,8 @@ of a large-scale computation, index, or analytic as new data is discovered. Chec
 ## Getting Started
 
 * Take the [Fluo Tour][tour] if you are completely new to Fluo.
-* Read the [install instructions][install] to install Fluo and start a Fluo application in YARN on a
-  cluster where Accumulo, Hadoop & Zookeeper are running. If you need help setting up these
+* Read the [Fluo documentation][fluo-docs] to learn how to install Fluo and start a Fluo application
+  on a cluster where Accumulo, Hadoop & Zookeeper are running. If you need help setting up these
   dependencies, see the [related projects page][related] for external projects that may help.
 
 ## Applications
```

Comment:
Thinking maybe should remove this section?

Reply:
I agree.  We should just point users to documentation on the website.